src/routes/about.tsx

```diff
@@ -0,0 +1,101 @@
+import Page from '@/components/page'
+import { css } from 'linaria'
+
+const pageStyle = css`
+  text-align: center;
+  padding: 0 1rem;
+  margin: auto;
+  max-width: 50rem;
```

Comment:
Would you consider reducing the max-width here to something a little narrower? Even though it will make the page taller, it will also make it easier to read.

A good typography rule of thumb that I usually use is to make the lines of text about 80 characters wide at the widest. That makes the text easier to scan and it makes it easier to find the next line when reading.

[Here's more about that](https://kickpoint.ca/the-readability-formula-making-your-website-easy-to-read/)